system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs

```diff
@@ -957,7 +957,7 @@ impl pallet_nfts::Config for Runtime {
 	type WeightInfo = weights::pallet_nfts::WeightInfo<Runtime>;
 	#[cfg(feature = "runtime-benchmarks")]
 	type Helper = ();
-	type BlockNumberProvider = System;
+	type BlockNumberProvider = RelaychainDataProvider<Runtime>;
```

Comment:
We need to check again what solution we settled upon to not make like hell for dApps.  
A dApp is just going to look at the delay in blocknumbers and not know whether it is RC or AH.  
We discussed this at some point in the channel, but I don't find it currently.

Reply:
Also the fact that we can accidentally use `DAYS` instead of `RC_DAYS` shows that its not typesafe, not even in the runtime. So surely not for dApps.